### ISSUE TITLE
Problem setting 'gpu_threads_conf'

### ISSUE THREAD

**Issue by jsmallz3** [fireice-uk/xmr-stak-nvidia#141]
Hi there,
I'm trying to set up a basic system using a single gtx 1070. When I go to launch the miner, I get the message:

'The configuration for 'gpu_threads_conf' in your config file is 'null'[...]'. 

I have copied and pasted the 'block within the asterisks' to my config.txt file, and beyond that tried to play with some other thread/block configurations that would fit the specifications seen in the README. I even tried rebuilding after making these changes but still am given the output that the 'gpu_threads_conf' is null.

I'm trying to figure if I am missing some important step in attempting to run this. Is there some re-compiling step or something that I am missing? Any help would be greatly appreciated.

I don't know if it matters but my CUDA is up to date.

**Comment by psychocrypt** [comment]
Use the [latest release](https://github.com/fireice-uk/xmr-stak-nvidia/releases) and use the default config, add only the pool address, username and password. The miner will show you the best config.

**Comment by jsmallz3** [comment]
I'm using the latest release, and have tried using the default config and the recommended one. I have populated the address, username/wallet address, and password appropriately, but am still facing this problem. 

My CUDA works, as I have been able to run the CUDA samples along with Tensorflow for GPU. I've tried playing some more with the config.txt file's GPU configuration a bit more, but to no avail.
